doc/Extensions/Services.md

```diff
@@ -210,7 +210,7 @@ These settings are related and should be investigated and set accordingly.
 The below values are not defaults or recommended.
 
 ```php
-$config['service_poller_enabled']           = true;
```

Comment:
Hmm, this section of the docs seems very unrelated to services.  Perhaps it should be trimmed down.

Reply:
Also, could swap to lnms config:set

Reply:
In this case I assumed that the documentation was left this way to be able to override this setting on a per node basis.

I think that some of the documentation around polling config including distributed polling may need a larger review.  For example the scaling document for distributed polling says that redis is preferred for locking, but the web UI, sample configuration and validation scripts only show how to configure memcached.  I think the service daemon may only use redis for queue management (but its documentation mentions memcached), while the cron wrappers only use memcache for device level locking.  There is also the mention of poller config in the services page above.

Reply:
I would say to leave it, but I think the change only adds to the existing confusion.